src/semantics.rs

```diff
@@ -123,11 +127,11 @@ impl Validate for Expression {
                         }
                     },
                     BinaryOperator::In | BinaryOperator::NotIn => {
-                        match p.rhs {
-                            Value::String(_) | Value::IpCidr(_) => {
+                        match (lhs_type, &p.rhs,) {
```

Comment:
Should be `(lhs_type, &p.rhs)`? (remove `,`)?

Reply:
This is by auto format, and is the recommended format for tuples by Rustfmt.